src/main/java/command/ByeCommand.java

```diff
@@ -0,0 +1,22 @@
+package command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Command to end the Duke program.
+ *
+ */
+public class ByeCommand extends Command {
+
+    public ByeCommand() {
+        this.setExit(true);
```

Comment:
I think it would be cleaner if the `ByeCommand` class simply `@Overrides` the `isExit` method (which by default would return `false`) from `Command` to return `true`, instead of maintaining and using an additional method `setExit` to set the parent's attribute.

Reply:
Yes I looked through your code and agree that it isn't necessary to really use a isExit variable since it is already unique to each command class hence just need to implement each isExit() method differently. Thank you for your feedback and idea!